Site/ASAB Maestro/Descriptors/jupyter.yaml

```diff
@@ -0,0 +1,115 @@
+define:
+  type: rc/descriptor
+  name: Jupyter Notebook
+
+secrets:
+  - JUPYTER_TOKEN
+
+descriptor:
+  user: "1000:0"
+  image: quay.io/jupyter/base-notebook
+
+  volumes:
+    - "{{SLOW_STORAGE}}/{{INSTANCE_ID}}:/home/jovyan"
+    - "{{SLOW_STORAGE}}/{{INSTANCE_ID}}/.jupyter:/home/jovyan/.jupyter"
+    - "{{SITE}}/{{INSTANCE_ID}}:/config"
+
+
+  entrypoint: ["/bin/bash", "-e", "/config/jupyter_start.sh"]
+
+files:
+  - "jupyter_notebook_config.py": |
+      c.ServerApp.token = "{{JUPYTER_TOKEN}}"
+      c.ServerApp.allow_origin = '*'
+      c.ServerApp.port = 8888
```

Comment:
Assign nicer port based on the well-known table.¨

ie 9002

Reply:
8888 is the Jupyter default port though, even wikipedia lists it as an unofficial well-known port, do we really want to overwrite it?

Reply:
meh ...

Reply:
so, I keep 8888